src/utils/utils.ts

```diff
@@ -393,3 +394,8 @@ parmcards4;
     `Please deploy the SASjs runner by running the code below and try again:\n${sasjsRunnerCode}`
   )
 }
+
+export const getAbsolutePath = (providedPath: string, relativePath: string) =>
```

Comment:
It would be good to see some tests for this.

Reply:
Added